tests/features/AddressTest.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+use Faker\Factory;
+use Tests\TestCase;
+use LaravelEnso\Core\app\Models\User;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Eloquent\Model;
+use LaravelEnso\Addresses\app\Models\Address;
+use \LaravelEnso\Addresses\app\Models\Country;
+use LaravelEnso\Addresses\app\Traits\Addressable;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Symfony\Component\HttpKernel\Exception\ConflictHttpException;
+
+class AddressTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private $user;
+    private $faker;
+    private $testModel;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+//      $this->withoutExceptionHandling();
+
+        $this->seed()
+            ->actingAs($this->user = User::first());
+
+        $this->createTestTable();
+
+        $this->faker = Factory::create();
+
+        $this->testModel = factory(Address::class)->create([
+            'addressable_id' => TestModel::create(['name' => 'addressable'])->id,
```

Comment:
We have to make the `TestModel` class name more specific because the file does not have a namespace and it will/can conflict with classes from other tests